script.js

```diff
@@ -1,105 +1,305 @@
+
+    const computerPaperPic = document.createElement("img");
+        computerPaperPic.src = "paper-23652_1280.png";
+        computerOptions.appendChild(computerPaperPic);
+        computerPaperPic.style.width = "100px";
+        computerPaperPic.style.height = "100px";
+        computerPaperPic.style.margin = "50px";
+
+    const computerScissorsPic = document.createElement("img");
+        computerScissorsPic.src = "scissors-2026687_1280.png";
+        computerOptions.appendChild(computerScissorsPic);
+        computerScissorsPic.style.width = "100px";
+        computerScissorsPic.style.height = "100px";
+        computerScissorsPic.style.margin = "50px";
+
+
+//create computer choice function 
 
 function getComputerChoice() {
     let randomValue = Math.floor(Math.random() * 3);
 
     if(randomValue === 0) {
+        computerRockPic.style.transform = "scale(1.5)";
+        computerPaperPic.style.transform = "scale(.5)";
+        computerScissorsPic.style.transform = "scale(.5)";
         return "rock";
 
     } else if(randomValue === 1) {
+        computerRockPic.style.transform = "scale(.5)";
+        computerPaperPic.style.transform = "scale(1.5)";
+        computerScissorsPic.style.transform = "scale(.5)";
         return "paper";
 
     } else {
+        computerRockPic.style.transform = "scale(.5)";
+        computerPaperPic.style.transform = "scale(.5)";
+        computerScissorsPic.style.transform = "scale(1.5)";
         return "scissors";
-    }
+    } 
 }
 
-//console.log(getComputerChoice(3));
+//create playRound function
 
-//function to get human choice with prompt
-// function to get human choice
-function getHumanChoice() {
-    let promptChoice = prompt("Rock, paper, or scissors?");
+function playRound(humanChoice, computerChoice) {
 
-        if(promptChoice.toLowerCase() === "rock") {
-            return "rock";
+    if (humanChoice === computerChoice) { 
+        winner.textContent = `It's a draw. Play again.`;
+    }
 
-        } else if(promptChoice.toLowerCase() === "paper") {
-            return "paper" 
+    else if (humanChoice === 'rock' && computerChoice === 'scissors' || humanChoice === 'paper' && computerChoice === 'rock' || humanChoice === 'scissors' && computerChoice === 'paper') {
+        winner.textContent = `${humanChoice} beats ${computerChoice}. You win!`;
+        humanScore+= 1;
+    }   
 
-        } else if(promptChoice.toLowerCase() === "scissors") {
-            return "scissors";
-        
-        } else {
-            console.log("Invalid choice. Please try again.");
-        }
-}
+    else { 
+        winner.textContent = `${computerChoice} beats ${humanChoice}. You lose.`;
+        computerScore+= 1;
+    }
 
-//console.log(getHumanChoice());
+} 
 
-const humanSelection = getHumanChoice();
-const computerSelection = getComputerChoice(3);
+function playGame() {
 
-//function for 5 round and end of game
+    if ((humanScore < 5) || (computerScore < 5)) {
```

Comment:
there is no way for the game to stop, since this only defines what should happen when the score is less than 5. The script doesn't know what to do when the score IS 5.

Issue: The check (humanScore < 5) || (computerScore < 5) does not stop the game when a player reaches 5 points.
Suggestion: Refactor playGame so that it only checks scores once per round, and declare a winner when either player reaches 5 points.

Reply:
Does the else statement that I have there not call the end of the game?  My idea was that if both scores are not less than 5, it should call the declare winner function for the end of the game, which I probably need to do some work with still.

   } else {
        buttons.remove();
        displayResults.remove();
        declareWinner();
    }
}


Also, couldn't figure out how you referenced lines like that...